### PR TITLE
Update validation.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
As I was reviewing the validation documentation I thought I'd try cleaning it up as I went along.

### CODE REVIEW

File: en/orm/validation.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ Validating Data Before Building Entities
 ========================================
 
 When marshalling data into entities, you can validate data. Validating data
-allows you to check the type, shape and size of data. By default request data
+allows you to check the type, shape, and size of data. By default request data
```
Comment (ravage84) [inline en/orm/validation.rst]: I doubt this is necessary, because of the "and".

File: en/orm/validation.rst [hunk 2]
```diff
@@ -26,17 +26,17 @@ fields with errors will not be present in the returned entity::
         // Entity failed validation.
     }
 
-When building an entity with validation enabled the following things happen:
+When building an entity with validation enabled the following occurs:
 
 1. The validator object is created.
 2. The ``table`` and ``default`` validation provider are attached.
-3. The named validation method is invoked. For example, ``validationDefault``.
+3. The named validation method is invoked (e.g., ``validationDefault``).
```
Comment (ravage84) [inline en/orm/validation.rst]: The comma is not necessary here, too.

File: en/orm/validation.rst [hunk 3]
```diff
@@ -84,13 +84,13 @@ To create a default validation object in your table, create the
         }
     }
 
-The available validation methods and rules come from the ``Validator`` class and
-they are documented in the :ref:`creating-validators` section.
+The available validation methods and rules come from the ``Validator`` class and
```
Comment (ravage84) [inline en/orm/validation.rst]: Please don't include trailing spaces.

File: en/orm/validation.rst [hunk 4]
```diff
@@ -84,13 +84,13 @@ To create a default validation object in your table, create the
...
-The available validation methods and rules come from the ``Validator`` class and
-they are documented in the :ref:`creating-validators` section.
+The available validation methods and rules come from the ``Validator`` class and 
+are documented in the :ref:`creating-validators` section.
 
 .. note::
 
-    Validation objects are mainly meant to be used to validate user input, i.e.
-    forms and any other posted request data.
+    Validation objects are intended primarily for validating user input (i.e.,
```
Comment (ravage84) [inline en/orm/validation.rst]: You seem to be a fan of brackets. Are you sure they are an improvement to the language of the book?
Normally they don't really help the flow of the sentences...

File: en/orm/validation.rst [hunk 5]
```diff
@@ -302,20 +302,23 @@ class::
 
         // Add a rule for create.
         $rules->addCreate(function ($entity, $options) {
+            // Return a boolean to indicate pass/fail
```
Comment (ravage84) [inline en/orm/validation.rst]: "failure", may be?

File: en/orm/validation.rst [hunk 6]
```diff
@@ -438,10 +441,10 @@ Validation is defined using the ``validationCustomName()`` methods::
         return $validator;
     }
 
-Validation is meant for forms, and request data. This means that validation rule
-sets can assume things about the structure of a form, and validate fields not in
-the schema of the database. Validation assumes strings or array are passed as
-that is what is received from any request::
+Validation is meant for forms and request data. This means that validation rule
+sets can assume things about the structure of a form and validate fields not in
+the schema of the database. Validation assumes strings or arrays are passed
```
Comment (ravage84) [inline en/orm/validation.rst]: Isn't it "an array" in that case?

**Comment by ravage84** [comment]
@jeffblack360 many thanks for your improvements! Don't rush to implement my critique. You can wait for other opinions.

**Comment by antograssiot** [comment]
:+1:

**Pull request merged**